ibis/backends/exasol/__init__.py

```diff
@@ -382,31 +384,27 @@ def create_table(
         else:
             temp_name = name
 
-        table = sg.table(temp_name, catalog=database, quoted=quoted)
-        target = sge.Schema(this=table, expressions=column_defs)
+        table_expr = sg.table(temp_name, catalog=database, quoted=quoted)
```

Comment:
I changed these variable names because `table = <not the memtable>` causes it to get GC'd. I gave another example of this in the PR description.